WolvenKit/Views/Dialogs/Windows/SelectFacialAnimationPathDialog.xaml

```diff
@@ -44,13 +44,15 @@
                     HorizontalAlignment="Stretch"
                     VerticalAlignment="Stretch">
                     <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="120" />
+                        <ColumnDefinition Width="140" />
                         <ColumnDefinition Width="300" />
                     </Grid.ColumnDefinitions>
                     <Grid.RowDefinitions>
                         <RowDefinition Height="Auto" />
                         <RowDefinition Height="Auto" />
                         <RowDefinition Height="Auto" />
+                        <RowDefinition Height="Auto" />
```

Comment:
Can you add extra rows in between fields, with a height of 5 as a margin to improve UX?

Reply:
Done and done